pkg/cvo/sync_worker.go

```diff
@@ -307,11 +308,16 @@ func (w *SyncWorker) Start(ctx context.Context, maxWorkers int) {
 				w.lock.Unlock()
 				defer cancelFn()
 
+				config, err := lister.Get(cvoOptrName)
+				if err != nil {
+					klog.Errorf("Failed to get cluster version data from ClusterVersionLister: %v", err)
```

Comment:
do we need to log this here?  I'd rather log it up the stack wherever we decide to ignore the returned `err` instead of exiting.

Reply:
Not clear to me what you're after here wrt "instead of exiting".

Reply:
Fixed in [64e0aba](https://github.com/openshift/cluster-version-operator/pull/364/commits/64e0aba9f9748579edc3988458e0b060ef1736c9)